src/eliater/examples/ecoli.py

```diff
@@ -1,4 +1,14 @@
-"""Examples for transcriptional Escherichia coli K-12 regulatory network."""
+"""Examples for transcriptional Escherichia coli K-12 regulatory network.
+
+The E. Coli regulatory network was extracted manually (by hand) from the EcoCyc
+database [Keseler2021]_ . The nodes represent genes, and the edges represent
+regulatory relationships.
+
+.. [Keseler2021] Keseler, I. M., Gama-Castro, S., Mackie, A., Billington, R.,
```

Comment:
all that actually matters is a DOI so people can follow the reference, please add this in